pipeline/src/node_properties/stream_rtsp_out_properties.rs

```diff
@@ -1,25 +1,26 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use url::Url;
 
-// Ditch all manual Serialize + Deserialize code after changing the properties to specific types.
-// This includes the use of `serialize_with` attribute.
-
-#[derive(Debug, Clone, PartialEq, Serialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct StreamRtspOutProperties {
```

Comment:
If this only consists of one member with `#[serde(flatten)]`, why do we need it at all? Are we planning to add more fields or should it maybe be a type alias instead?

Reply:
It is for consistency. Everything added at runtime goes under `runtime` field. Just wanted to make it the same way for all node types.